### DISCUSSION TITLE
Allow us to customise the Home menu tab

### DISCUSSION THREAD

**Discussion by KillerParticles** [ciderapp/Cider#681] [Feature Request]
![image](https://user-images.githubusercontent.com/93286201/159814821-2d7c0a9b-64a4-48aa-bc6f-6a95bf75e280.png)

This home menu currently cannot be customised with whats on it, allow us to customise what we can put on it. For example let me replace the recently played songs with recently played albums instead, like how Marvis Pro IOS app does it.

**Comment by ghost** [comment]


**Reply by ghost** [reply]
https://github.com/ciderapp/Cider/discussions/681

**Reply by ghost** [reply]
> https://github.com/ciderapp/Cider/discussions/681

**Comment by ghost** [comment]
> ![image](https://user-images.githubusercontent.com/93286201/159814821-2d7c0a9b-64a4-48aa-bc6f-6a95bf75e280.png)
> 
> 
> 
> This home menu currently cannot be customised with whats on it, allow us to customise what we can put on it. For example let me replace the recently played songs with recently played albums instead, like how Marvis Pro IOS app does it.

**Reply by ghost** [reply]
https://github.com/ciderapp/Cider/discussions/681
